samples/net/secure_mqtt_sensor_actuator/sample.yaml

```diff
@@ -0,0 +1,22 @@
+sample:
+  name: Secure MQTT Sensor/Actuator Sample
+tests:
+  sample.net.secure_mqtt_sensor_actuator:
+    harness: net
+    tags:
+      - net
+      - mqtt
+      - sensors
+    filter: dt_alias_exists("ambient-temp0")
+    integration_platforms:
+      - adi_eval_adin1110ebz
+  sample.net.secure_mqtt_sensor_actuator.staticip_insecure:
```

Comment:
And finally, would be good to add one build configuration for `native_sim` for example.

Reply:
Done